### ISSUE TITLE
[Kura 2.1.0 QA] GPIO example bundle configuration

### ISSUE THREAD

**Issue by markoer** [eclipse/kura#893]
PTC-GPIO-003 from #788 envisages a few configuration parameters to be set to different values and stored. What actually happens and log and page reload confirm it is that only pins are stored and the rest of the configuration is discarded:

```
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent - Acquiring GPIO pin 17 with params:
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Direction....: 3
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Mode.........: -1
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Trigger......: -1
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent - GPIO pin 17 acquired
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent - Acquiring GPIO pin 18 with params:
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Direction....: 3
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Mode.........: -1
[Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Trigger......: -1
```

GPIO bundle from release-2.1.0 branch, commit e38afd4

**Comment by djurod** [comment]
I had the same kind of issues with BeagleBone Black on build 15:

```
2016-12-09 14:18:40,409 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent - Acquiring GPIO pin 45 with params:
2016-12-09 14:18:40,410 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Direction....: 3
2016-12-09 14:18:40,410 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Mode.........: -1
2016-12-09 14:18:40,411 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Trigger......: -1
2016-12-09 14:18:40,412 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent - GPIO pin 45 acquired
2016-12-09 14:18:40,418 [pool-22-thread-1] INFO  o.e.k.e.g.GpioComponent - Setting GPIO pin JdkDioPin [pinIndex=45, pinName=GPIO_45] to true
2016-12-09 14:18:40,419 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent - Acquiring GPIO pin 44 with params:
2016-12-09 14:18:40,420 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Direction....: 3
2016-12-09 14:18:40,420 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Mode.........: -1
2016-12-09 14:18:40,420 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent -    Trigger......: -1
2016-12-09 14:18:40,422 [Component Resolve Thread (Bundle 7)] INFO  o.e.k.e.g.GpioComponent - GPIO pin 44 acquired
2016-12-09 14:18:40,432 [pool-22-thread-1] INFO  o.e.k.e.g.GpioComponent - Setting GPIO pin JdkDioPin [pinIndex=44, pinName=GPIO_44] to true
```

**Comment by cdealti** [comment]
Closed via #949.

**Comment by markoer** [comment]
Verified on Kura 2.1.0 build 19.
